fusor-ember-cli/app/controllers/assign-nodes.js

```diff
@@ -286,10 +272,8 @@ export default Ember.Controller.extend(
           "X-CSRF-Token": token
         }
       }).catch(function (error) {
-        error = error.jqXHR;
         console.log('ERROR');
         console.log(error);
-        self.set('showLoadingSpinner', false);
```

Comment:
Looks like the spinner doesn't get shown anywhere, so this was removed since it does nothing?